theme/static/js/custom.js

```diff
@@ -157,6 +157,44 @@ $(document).ready(function () {
         return false
     });
 
+    $("#hs-nav-bar .res-dropdown ul > li>  a").on("click", function () {
+        var formData = new FormData();
+
+        formData.append("csrfmiddlewaretoken", csrf_token);
```

Comment:
@pkdash It would be good to get rid of these values from the backend if they are not needed in the requests anymore. I've added them here for now because without them the request doesn't go through.

Reply:
We could add these defaults server side and resource-type default to composite. making it so you only have to pass the token.  Actually with this new workflow, I think we could probably remove all of the irods parameters completely from the create method.  @hyi Is there any reason we can't remove the irods paramaters from "/hsapi/_internal/create-resource/do/"?

Reply:
@pkdash @Maurier @sblack-usu These iRODS-related parameters in the request are for files selected from iRODS file browser when creating a resource using files from iRODS user zone or other third party iRODS zone, so we cannot get rid of them.

Reply:
@hyi - But with the new create resource workflow, these parameters will never be used.

Reply:
@sblack-usu OK. That makes sense. Then we can remove those parameters, or at least add a default when retrieving those request data, such as this line: https://github.com/hydroshare/hydroshare/blob/develop/hs_core/views/__init__.py#L1221 @Maurier @pkdash